core/iwasm/libraries/lib-socket/src/wasi/wasi_socket_ext.c

```diff
@@ -142,7 +142,7 @@ accept(int sockfd, struct sockaddr *addr, socklen_t *addrlen)
 
     memset(&wasi_addr, 0, sizeof(wasi_addr));
 
-    error = __wasi_sock_accept(sockfd, 0, &new_sockfd);
+    error = __wasi_sock_accept(sockfd, &new_sockfd);
```

Comment:
Should not change this line, refer to: https://github.com/bytecodealliance/wasm-micro-runtime/pull/1531